### ISSUE TITLE
Create a New Medusa Project

### ISSUE THREAD

**Issue by warrenbeufesche** [medusajs/medusa#5414]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

Create a New Medusa Project section does not work on Ubuntu 22 and PostgreSQL and seems like it may be a problem on other systems. Below is the command I used to install. This was mentioned in GitHub Discussions. I almost gave up after many hours, glad I did not. This seems like a pretty huge problem for those new to Medusa, but maybe I am missing something?

npx create-medusa-app@latest --db-url postgres:://user:password@localhost:5432/giftstore

### How can this issue be resolved?

1. Update docs
2.
3.
...

### Are you interested in working on this issue?

- [X] I would like to fix this issue

**Comment by shahednasser** [comment]
Thank you for opening this issue @warrenbeufesche . Can you please clarify what is the exact issue/error you face when you run the above command?

**Comment by warrenbeufesche** [comment]
I get message: ALERT: Host information must be supplied. I did not know what the string should be, until I found the GitHub Discussion (Thank you for that!). This was from an install about Oct 16. I think there is a new update, maybe that fixes it. I have not tried the new update. There must be something about my Ubuntu 22.04 that causes the problem, otherwise you would see this all the time. It is working now for me but it seems odd that the installation instructions did not work.

**Comment by shahednasser** [comment]
Just to clarify, you first used:

```
npx create-medusa-app@latest
```

And got the error `ALERT: Host information must be supplied.`

Then tried:

```
npx create-medusa-app@latest --db-url postgres:://user:password@localhost:5432/giftstore
```

And it worked, correct?

I will investigate the issue and see if there's a way to resolve it, but would be great if you could confirm the above.

**Comment by warrenbeufesche** [comment]
So sorry, user error. Yes the second command worked, and now I have the first command working. I was using the Ubuntu user password. I did not realize the PostgreSQL role postgres has the password: postgres by default. When I changed the postgres user password, everything worked fine. In my defense I spent many hours with many AIs (BIG mistake). I should have search the old fashion way. I will take a 40 hour Postgres class before asking any more questions. :)
